.pre-commit-config.yaml

```diff
@@ -15,20 +15,9 @@ repos:
     # Ruff version.
     rev: 'v0.6.3'
     hooks:
+      - id: ruff-format
       - id: ruff
         args: ["--fix", "--show-fixes"]
-  # https://github.com/python/black#version-control-integration
-  - repo: https://github.com/psf/black-pre-commit-mirror
-    rev: 24.8.0
-    hooks:
-      - id: black-jupyter
-  - repo: https://github.com/keewis/blackdoc
-    rev: v0.3.9
-    hooks:
-      - id: blackdoc
-        exclude: "generate_aggregations.py"
```

Comment:
We should probably add the generation scripts to ruff as well (in the pyproject.toml?)